src/main/java/com/codegym/jrugotom5/service/AdvertService.java

```diff
@@ -30,4 +30,8 @@ public List<AdvertDTO> getAdvertsByDateRange(LocalDate from, LocalDate to) {
                 .map(advert -> modelMapper.map(advert, AdvertDTO.class))
                 .collect(Collectors.toList());
     }
+    public AdvertDTO update(AdvertDTO advertDTO){
+        Advert advert = modelMapper.map(advertDTO, Advert.class);
```

Comment:
this mapping will not work correctly without custom converter from AdvertDTO to Advert